Engine/DataFeeds/SubscriptionData.cs

```diff
@@ -72,6 +80,41 @@ public static SubscriptionData Create(SubscriptionDataConfig configuration, Secu
                 data.Time = data.Time.ExchangeRoundDownInTimeZone(configuration.Increment, exchangeHours, configuration.DataTimeZone, configuration.ExtendedMarketHours);
             }
 
+            if (factor.HasValue && (factor.Value != 1 || configuration.SumOfDividends != 0))
```

Comment:
Let's simplify this behavior so we only support for now going from Any data mode -> Raw and other cases throw invalid operation exception 👍